tests/themis/themis_secure_session.c

```diff
@@ -668,11 +666,40 @@ static void test_basic_flow_no_transport(void)
     }
 }
 
+static void test_empty_peer_id(void)
+{
+    memcpy(&(client.transport), &transport, sizeof(secure_session_user_callbacks_t));
+    client.transport.user_data = &client;
+
+    client.session =
+        secure_session_create(client.id, 0, client.priv, client.priv_length, &(client.transport));
+
+    testsuite_fail_if(client.session != NULL, "secure_session_create must fail with empty Peer ID");
+}
+
+static void test_invalid_private_key_type(void)
```

Comment:
test public key?

Reply:
IMO, it's would be better to split this into separate tests: RSA private keys, RSA public keys, EC public keys.

Reply:
agree